android/app/.project

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It looks like this file was included unintentionally.

Want to add a line for it to our gitignore? Probably it should go in this block:
```
# VS Code: default Java extension
/android/.project
/android/.settings/
```
which already has a similar line. Though if that comment doesn't describe how you ended up with this file, then we'd update the comment too.